Grafana UI: V9 beta packages cause plugin tests to fail

**What happened**:

Can't build custom data sources and application plugins using Grafana 9.0.0-beta3

```
yarn run v1.22.17
$ grafana-toolkit plugin:build --coverage
  Using Node.js v16.15.0
  Using @grafana/toolkit v9.0.0-beta3
✔ Preparing
✔ Linting
 FAIL  src/components/RootPage/RootPage.test.tsx
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      src/components/RootPage/RootPage.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)

 FAIL  src/components/Config/Config.test.tsx
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      node_modules/@grafana/runtime/index.development.js
      node_modules/@grafana/runtime/index.js
      src/components/Config/Config.tsx
      src/components/Config/Config.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)

 FAIL  src/module.test.ts
  ● Test suite failed to run

    Cannot find module 'indexof' from 'node_modules/@grafana/ui/index.development.js'

    Require stack:
      node_modules/@grafana/ui/index.development.js
      node_modules/@grafana/ui/index.js
      node_modules/@grafana/runtime/index.development.js
      node_modules/@grafana/runtime/index.js
      src/components/Config/Config.tsx
      src/components/Config/index.ts
      src/components/index.ts
      src/module.ts
      src/module.test.ts

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (node_modules/@grafana/ui/index.development.js:11:20)
```

**What you expected to happen**:

Build successful.

**How to reproduce it (as minimally and precisely as possible)**:

1. Download https://github.com/VolkovLabs/volkovlabs-abc-datasource
2. Remove `indexof` from `package.json`, `yarn install`
3. Try to build

**Anything else we need to know?**:

Adding `indexof` to the `package.json` resolves the issue.

**Environment**:
- Grafana version: 9.0.0-beta3
- Data source type & version: Custom Template
- OS Grafana is installed on: 9.0.0-beta3
- User OS & Browser: N/A
- Grafana plugins:
- Others:

--- a/packages/grafana-ui/package.json
+++ b/packages/grafana-ui/package.json
@@ -170,7 +170,6 @@
     "rollup-plugin-sourcemaps": "0.6.3",
     "rollup-plugin-svg-import": "^1.6.0",
     "rollup-plugin-terser": "7.0.2",
-    "rollup-plugin-visualizer": "^5.6.0",
     "sass-loader": "13.0.0",
     "storybook-dark-mode": "1.1.0",
     "style-loader": "3.3.1",

--- a/packages/grafana-ui/rollup.config.ts
+++ b/packages/grafana-ui/rollup.config.ts
@@ -53,4 +53,4 @@ const buildCjsPackage = ({ env }) => {
     ],
   };
 };
-export default [buildCjsPackage({ env: 'development' })];
+export default [buildCjsPackage({ env: 'development' }), buildCjsPackage({ env: 'production' })];

